### PR TITLE
Updates README.md

### PR DESCRIPTION
**Pull request by AlphaMale1996**
- Adds Aayushman's details

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ To complete this exercise you need to:
   You should also go through the [curated list of git
 resource](https://help.github.com/articles/good-resources-for-learning-git-and-github/) provided by github to find other resouces to fit your need.
 
-- **Fork the repo, create a new branch, make changes and send a pull request**
+- **Fork the repo, create a new branch, make changes and send a pull request**/
```
Comment (hargup) [inline README.md]: Please fix the change made in this line.

**Comment by AlphaMale1996** [comment]
I have rebased and fixed the commit.
@icyflame

**Comment by hargup** [comment]
:+1:

**Comment by icyflame** [comment]
@hargup The README says that entry must maintain the alphabetical order. There was a revision required for this PR to be merge-able.

**Comment by hargup** [comment]
Oops I wasn't careful enough. @AlphaMale1996 I've reverted the merge, please fix the issue and inform us.

**Pull request merged**